src/diff/index.js

```diff
@@ -381,8 +395,20 @@ function catchErrorInComponent(error, component) {
 			}
 			catch (e) {
 				error = e;
+				isSuspend = false;
 			}
 		}
 	}
+
+	// TODO: Add a react-like error message to preact/debug
```

Comment:
Does this Todo need to be resolved ?
If it is unnecessary, it is better to remove it.

Reply:
I'd like to tackle this (see TODO in PR description) but currently don't know how to to this without adding too many bytes to core